models/garant-environement.ts

```diff
@@ -0,0 +1,65 @@
+import { clientRGE, WORKING_WITH_ENUM } from '#clients/rge';
+import { EAdministration } from '#models/administrations';
+import {
+  APINotRespondingFactory,
+  IAPINotRespondingError,
+} from '#models/api-not-responding';
+import { Siret, verifySiren } from '#utils/helpers';
+import logErrorInSentry from '#utils/sentry';
+
```

Comment:
Client seems good enough at this stage. 

In order to make it evolve I think we will eventually need an aditionnal layer of abstraction with a macro model for certifcations and sub models for each certificate.

We can refacto later

```
models/certifications
models/certifications/rge
```

Reply:
That a good point let's do it to futurproof